README.md

```diff
@@ -1,33 +1,195 @@
+algo.set_option('threads')
+algo.set_option('max_lhs')
+algo.set_option('seed')
+algo.execute()
+result = algo.get_fds()
+print('AFDs:')
+for fd in result:
+	print(fd)
+```
+```text
+>>> AFDs:
+>>> ( 0 ) -> 1
+>>> ( 0 ) -> 2
+>>> ( 1 ) -> 2
+```
+
+3) Check whether metric functional dependency “Title -> Duration” with radius 5 (using the Euclidean metric) holds in a table represented by a .csv file that uses a comma as the separator and has a header row. In this example the default MFD validation algorithm (BRUTE) is used.
+
+```python
+import desbordante
+
+TABLE = '../examples/datasets/theatres_mfd.csv'
+METRIC = 'euclidean'
+LHS_INDICES = [0]
+RHS_INDICES = [2]
+PARAMETER = 5
+
+algo = desbordante.MetricVerifier()
+algo.set_option('table', (TABLE, ',', True))
+algo.set_option('is_null_equal_null')
+algo.load_data()
+algo.set_option('lhs_indices', LHS_INDICES)
+algo.set_option('metric', METRIC)
+algo.set_option('parameter', PARAMETER)
+algo.set_option('dist_from_null_is_infinity')
+algo.set_option('rhs_indices', RHS_INDICES)
+algo.execute()
+if algo.mfd_holds():
+    print('MFD holds')
+else:
+    print('MFD not holds')
+```
+```text
+>>> MFD holds
+```
+4) Discover approximate functional dependencies with various error thresholds. Here, we showcase a preferred approach to configuring algorithm options. Furthermore, we are using a pandas dataframe to load data from a CSV file.
+```python-repl
+>>> import desbordante
+>>> import pandas as pd
+>>> pyro = desbordante.Pyro()
+>>> df = pd.read_csv('iris.csv', sep=',', header=0)
+>>> pyro.load_data(df)
+>>> pyro.execute(error=0.0)
+>>> pyro.get_fds()
+[( 0 1 2 ) -> 4, ( 0 2 3 ) -> 4, ( 0 1 3 ) -> 4, ( 1 2 3 ) -> 4]
+>>> pyro.execute(error=0.1)
+>>> pyro.get_fds()
+[( 2 ) -> 0, ( 2 ) -> 1, ( 0 ) -> 2, ( 2 ) -> 4, ( 2 ) -> 3, ( 3 ) -> 2, ( 3 ) -> 0, ( 0 ) -> 1, ( 0 ) -> 3, ( 1 ) -> 0, ( 1 ) -> 2, ( 3 ) -> 4, ( 3 ) -> 1, ( 1 ) -> 3, ( 0 ) -> 4, ( 1 ) -> 4]
+>>> pyro.execute(error=0.2)
+>>> pyro.get_fds()
+[( 2 ) -> 1, ( 2 ) -> 0, ( 2 ) -> 4, ( 0 ) -> 2, ( 2 ) -> 3, ( 0 ) -> 1, ( 3 ) -> 4, ( 3 ) -> 2, ( 3 ) -> 1, ( 3 ) -> 0, ( 1 ) -> 2, ( 0 ) -> 3, ( 0 ) -> 4, ( 1 ) -> 0, ( 1 ) -> 4, ( 1 ) -> 3]
+>>> pyro.execute(error=0.3)
+>>> pyro.get_fds()
+[( 2 ) -> 1, ( 0 ) -> 2, ( 2 ) -> 0, ( 3 ) -> 0, ( 2 ) -> 3, ( 1 ) -> 0, ( 2 ) -> 4, ( 3 ) -> 2, ( 0 ) -> 1, ( 1 ) -> 2, ( 3 ) -> 1, ( 3 ) -> 4, ( 0 ) -> 3, ( 4 ) -> 2, ( 4 ) -> 1, ( 0 ) -> 4, ( 1 ) -> 3, ( 1 ) -> 4, ( 4 ) -> 3]
+```
+
+## Web interface
+
+While the Python interface makes building interactive applications possible, Desbordante also offers a web interface which is aimed specifically for interactive tasks. For pattern discovery/validation a user may specify parameters, browse results, employ advanced visualizations and filters, all in a convenient way. Furthermore, we manually select the most interesting Python scenarios and implement them inside our web version. Currently, only the typo detection scenario is implemented.
```

Comment:
```suggestion
While the Python interface makes building interactive applications possible, Desbordante also offers a web interface which is aimed specifically for interactive tasks. For pattern discovery/validation a user may specify parameters, browse results, employ advanced visualizations and filters, all in a convenient way. Furthermore, we select the most interesting Python scenarios and implement them in the web version. Currently, only the typo detection scenario is implemented.
```
What are "interactive tasks"?

> For pattern discovery/validation a user may specify parameters, browse results, employ advanced visualizations and filters

Those seem like separate things, how are they connected? Also, specifying parameters and browsing results seems like basic functionality for a web app like this, not something worth talking up.